src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -12,68 +13,400 @@
 import teammates.common.datatransfer.FeedbackSessionAttributes;
 import teammates.common.datatransfer.InstructorAttributes;
 import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.util.Assumption;
 import teammates.common.util.Const;
+import teammates.common.util.TimeHelper;
+import teammates.common.util.Url;
+import teammates.ui.template.AdditionalSettingsFormSegment;
+import teammates.ui.template.ElementTag;
+import teammates.ui.template.FeedbackQuestionCopyTable;
+import teammates.ui.template.FeedbackQuestionEditForm;
+import teammates.ui.template.FeedbackQuestionGeneralSettings;
+import teammates.ui.template.FeedbackQuestionTableRow;
+import teammates.ui.template.FeedbackSessionPreviewForm;
+import teammates.ui.template.FeedbackSessionsForm;
 
 public class InstructorFeedbackEditPageData extends PageData {
 
-    public FeedbackSessionAttributes session;
-    public FeedbackQuestionAttributes newQuestion;
-    public List<FeedbackQuestionAttributes> questions;
-    public List<FeedbackQuestionAttributes> copiableQuestions;
-    public Map<String, Boolean> questionHasResponses;
-    public List<StudentAttributes> studentList;
-    public List<InstructorAttributes> instructorList;
-    public InstructorAttributes instructor;
-
+    private static final String SAVE_CHANGES = "Save Changes";
```

Comment:
This appears to be used only once.
It may be better to just use the String directly instead of leaving it here without any context.